src/components/ColorSwatchGrid.js

```diff
@@ -1,80 +1,17 @@
 import React from 'react';
-import { StaticQuery, graphql } from 'gatsby';
-import ColorSwatch from './ColorSwatch';
+import PropTypes from 'prop-types';
 
-const ColorSwatchGrid = ({ color, additionalClasses }) => (
-  <StaticQuery
-    query={
-        graphql`
-          query SassColors {
-            allSassVarsJson {
-              edges {
-                node {
-                  description
-                  context {
-                    name
-                    value
-                  }
-                  file {
-                    name
-                  }
-                  group
-                  type
-                }
-              }
-            }
-          }
-    `}
-    render={(data) => {
-      const sassColorData = data.allSassVarsJson.edges
-        .filter(item => {
-          return (
-            item.node.file.name === '_colors.scss' &&
-            item.node.group[0] === color
-          );
-        })
-        .map(item => (
-          {
-            name: item.node.context.name,
-            value: item.node.context.value,
-            description: item.node.description,
-            hexValue: item.node.type,
-          }
-        ));
+const SwatchGrid = ({ children, additionalClasses }) => {
+  return (
+    <div className={['docs-o-ColorSwatchGrid', additionalClasses].join(' ')}>
```

Comment:
i think you'd need to spread the addtionalClasses array into this one

Reply:
fixed